vmdb/app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -20,26 +20,37 @@ def save_inventory_multi(type, parent, hashes, deletes, find_key, child_keys = [
   end
 
   def save_inventory_single(type, parent, hash, child_keys = [], extra_keys = [])
-    find_key, child_keys, extra_keys, remove_keys = self.save_inventory_prep(nil, child_keys, extra_keys)
+    child_keys, extra_keys, remove_keys = self.save_inventory_prep(child_keys, extra_keys)
```

Comment:
if we didn't have prep, this would read:

```ruby
find_keys = Array.wrap(find_keys)
child_keys = Array.wrap(child_keys)
remove_keys = Array.wrap(extra_keys) + child_keys
# or get rid of new variable remove_keys and just rename it to extra_keys
```

Then the single case would be:

```ruby
child_keys = Array.wrap(child_keys)
remove_keys = Array.wrap(extra_keys) + child_keys
```